udfs/remote_udfs/nlp/README.md

```diff
@@ -0,0 +1,129 @@
+# Calling Google's Natural Language API Sentiment Analysis Example 
+
+Google's Natural Language API Sentiment Analysis inspects the given text and identifies the prevailing emotional opinion within the text, especially to determine a writer's attitude as positive, negative, or neutral.  
+
+To run this example, you will need the following APIs enabled:
+* [BigQuery API](https://cloud.google.com/bigquery/docs)
+* [Cloud Functions API](https://cloud.google.com/functions/docs)
+* [BigQuery Connections API](https://cloud.google.com/bigquery/docs/working-with-connections#enable_the_connection_service)
+* [Natural Language API](https://cloud.google.com/natural-language/docs/setup#api)
+
+## Running the example
+
+### Setting the environment variables 
+
+Replace the various environment variables below with your desired values.
+```
+CONNECTION_NAME=%Functional BigQuery Connection name% 
+DISPLAY_NAME=%Friendly or Display BigQuery Connection name%
+PROJECT=%Your project name% 
+LOCATION=%Your BigQuery dataset location% 
+CF_NAME=%Your Cloud Function name%
+DATASET=%Your BigQuery Dataset to deploy to%
```

Comment:
Since this is an example, you can greatly simplify it by only having users choose:

- LOCATION
- PROJECT
- DATASET

Provide static values for the rest.